kornia/filters/bilateral.py

```diff
@@ -0,0 +1,63 @@
+import math
+
+import numpy as np
+
+
+# gaussian filter
+def gfunc(x,y,sigma):
```

Comment:
No need in implementing this, we have gaussian2d kernel https://github.com/kornia/kornia/blob/master/kornia/filters/kernels.py#L424

Reply:
> No need in implementing this, we have gaussian2d kernel https://github.com/kornia/kornia/blob/master/kornia/filters/kernels.py#L424

I have a doubt. The current gaussian kernel available with Kornia, is it distance dependent? Because we need 2 gaussian functions, intensity difference dependent and distance difference dependent. So I dont think we can use the current gaussian2d kernel. Because it does not factor in the difference of values.

Reply:
At least this function does not depend on distance :) Other - yes

Reply:
> At least this function does not depend on distance :) Other - yes

I mean we have to calculate the gaussian of the distance of intensity the current pixel from the target pixel. The current kernel directly squares the values, as far as I understood the implementation.

Reply:
this could be a potential separated functionality